docs/how_to/use_optimizationlevel.rst

```diff
@@ -0,0 +1,276 @@
+#####################################
+Setting transpiler optimization level
+#####################################
+
+This guide shows you how to use the ``optimization_level``
+parameter of :func:`~qiskit.compiler.transpile`.
```

Comment:
```suggestion
This guide describes the effects of the ``optimization_level``
parameter of :func:`~qiskit.compiler.transpile`.
```
Just a suggestion; Using the parameter is easy (just changing one character), I think this page is more describing / explaining the behavior.